### PR TITLE
Add 0301 and 1008 options, fix max usdc pool balance

### PR DESCRIPTION
**Pull request by tensojka**
This is Proposal 19 and adds options that expire on the 3rd and 10th of August. It also fixes the wrongly too highly set usdc liquidity pool balance – this sets it to 50k as previously accepted by governance. This is for security reasons.

Class hash 0x624711767220c18293111756f63cf61394bc7dff690976e7f666d65ba958f89

### CODE REVIEW

**Review by Chepelau — APPROVED** [review]

**Comment by tensojka** [comment]
Added options that expire on 24th August as requested by users and increased voting time (applicable from next proposal) to 1500 to reflect the faster Starknet

New class hash 0xa8ee1fdf54164d651ced5b47032206c2aff48c54ccf7a8bab4a3f677000038

File: src/options.cairo [hunk 1]
```diff
@@ -457,4 +458,148 @@ mod Options {
 
         add_options(1008235894263284012, to_add.span())
     }
+
+    fn add_2408_options() {
+        let MATURITY: felt252 = 1691711999;
```
Comment (Chepelau) [inline src/options.cairo:463]: The maturity is 10/08, not 24/08

Comment (tensojka) [reply src/options.cairo]: Fixed

**Comment by Chepelau** [comment]
Otherwise lgtm

**Comment by tensojka** [comment]
New class hash 0x2c0e4d73f6e4a7a7530e3836396cd1ffced98549a81fd7a81146a7a3a428342

**Pull request closed without merging**